lib/fog/storage/openstack/requests/delete_container.rb

```diff
@@ -1,6 +1,12 @@
 module Fog
   module Storage
     class OpenStack
+      class Mock
+        def delete_container(name)
+          fail "Mock Not Implemented (#delete_container) in: #{__FILE__}:#{__LINE__}"
```

Comment:
I know you did a copy and paste but you dont need to define this. Once It is not implemented fog will throw the error automatically. =)

So you can skip this change.